packages/components/pie-checkbox/test/component/pie-checkbox.spec.ts

```diff
@@ -17,4 +24,194 @@ test.describe('PieCheckbox - Component tests', () => {
         // Assert
         expect(checkbox).toBeVisible();
     });
+
+    test.describe('Props', () => {
+        test.describe('value', () => {
+            test('should default to "on" string if no value prop provided', async ({ mount }) => {
+                // Arrange
+                const component = await mount(PieCheckbox, {});
+
+                // Act
+                const checkbox = component.locator(componentSelector);
+
+                // Assert
+                expect((await checkbox.inputValue())).toBe('on');
```

Comment:
Potentially stupid question – why would the default value be `on` and not `undefined` or `''` (empty) here?

Reply:
Not stupid at all, I forgot to add a note about it as I learned about it myself just while working on this component. It is a default value of `value` attr for html input type=checkbox so if it is not set by a user when the form is submitted with the checked checkbox the submitted value is "on". [The mdn doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value) mentioning it for the ref.

Reply:
Might be worth adding a comment into the test linking to that, as that's really useful to know 😄

Reply:
I updated the test and added a comment to the `expected` constant.